planning_interface/move_group_interface/include/moveit/move_group_interface/move_group.h

```diff
@@ -112,7 +112,10 @@ class MoveGroup
 
   /** \brief Construct a client for the MoveGroup action using a specified set of options \e opt. Optionally, specify a TF instance to use.
       If not specified, one will be constructed internally. A timeout for connecting to the action server can also be specified. If it is not specified,
-      the wait time is unlimited. */
+      the wait time is unlimited.
+    *
+    * @param opt. An option structure, if you pass a node with a specific callback queue, it has to of type ros::CallbackQueue.
+    */
```

Comment:
It's just to highlight that the callback queue must be of type ros::CallbackQueue. It's highly unlikely that it happens but it seems to be possible from the way it is coded.